static/src/js/components/OrderStatus/__tests__/OrderStatusItem.test.js

```diff
@@ -1942,6 +1942,129 @@ describe('OrderStatusItem', () => {
         expect(orderStatusTab.props().title).toEqual('Order Status')
         expect(orderStatusTab.childAt(0).props().orders).toEqual(props.collection.orders)
       })
+
+      test('renders an updated progress state', () => {
+        const { enzymeWrapper, props } = setup({
```

Comment:
why are you adding an enzyme test?

Reply:
Ensure that the Order status increments when harmony returns `complete_with_errors`. Or do you mean why not convert the whole file to RTL?

Reply:
Probably the latter...

Reply:
Yea this is too high of priority I think to do that now. Its gonna take significantly longer than the rest of the solution

Reply:
I think that's going to be more complex than the ticket given the priority of this item I think we should resolve that tech debt at another tim